corehq/apps/notifications/static/notifications/spec/NotificationsService.spec.mo.js

```diff
@@ -0,0 +1,62 @@
+describe('NotificationsService Unit Tests', function() {
+    var fakeRMIUrl = '/fake/rmi',
+        viewModel,
+        FakePromise = function (options) {
+            var self = this;
+            self.successCallback = function () {};
+            self.errorCallback = function () {};
+            self.mock = function (options) {
+                return {
+                    done: function (callback) {
+                        self.successCallback = callback;
+                        return fakePromise.mock(options);
+                    },
+                    fail: function (errorCallback) {
+                        self.errorCallback = errorCallback;
+                        return fakePromise.mock(options);
+                    }
+                };
+            };
+        };
+
+    var fakePromise = new FakePromise({});
+    sinon.stub(jQuery, 'ajax', fakePromise.mock);
+
+    it('Initialization', function () {
+        viewModel = $('#js-settingsmenu-notifications').startNotificationsService(fakeRMIUrl);
```

Comment:
i don't think this actually tests anything. mocha doesn't fail the test if there's javascript error